### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by XsidX**
For this project, I have:
-added a contact form
-added svg background for the hero and about sections
-changed body background color and added a white background color for the projects containers.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -326,6 +324,25 @@ <h1 class="intro-text">About<br>
...
+          If you have an application you are interested in developing,
+          a feature that you need built or a project that needs
+          coding. I’d love to help with it
+        </p>
+      <form action="https://formspree.io/f/mgedddej" method="post">
+        <ul>
+          <li><input name="Name" id="text" type="text" placeholder="Your name" maxlength="30" required></li>
+          <li><input name="Email" id="email" type="email" placeholder="Your email address" required></li>
+          <li><textarea name="comment" placeholder="Write your message here" maxlength="500" required></textarea></li>
+          <li><button type="submit" value="Submit" class="btn">Get In Touch</button></li>
```
Comment (VICTOR-CHUKWU) [inline index.html:341]: - [ ] Kindly make your submit button interactive with CSS pseudo classes like :hover
  you design :point_down: 
Screenshot from 2022-05-31 15-53-03

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @XsidX  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (VICTOR-CHUKWU) [reply index.html:341]: then add active class or visited or focus, I used hover as an example

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @XsidX 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**